### ISSUE TITLE
Laravel 5.8 support

### ISSUE THREAD

**Issue by antonkomarev** [cybercog/laravel-love#43]
Add support of Laravel 5.8

**Comment by hassanazimi** [comment]
Can you update in the Packagist as well, please?

**Comment by hassanazimi** [comment]
Version 6 completely changes and i cannot update to it.
Can you make sure you support 5.8 in your version 5 please!?

**Comment by antonkomarev** [comment]
@hassanazimi you are right. This package has changes a lot and it's completely another approach.
I see here 2 solutions:
1) Make a fork and maintain it by yourself
2) I can resurrect [laravel-likeable](https://github.com/cybercog/laravel-likeable) package, from which this one was born, but it will get only version support releases without any new features implementation. If anybody want to maintain and improve old implementation I can share an access to it.

**Comment by taliptako** [comment]
@hassanazimi I fork this project https://github.com/taliptako/laravel-love

**Comment by antonkomarev** [comment]
@taliptako you still have 6.x tags:
- in repository: https://github.com/taliptako/laravel-love/tags
- in packagist: https://packagist.org/packages/taliptako/laravel-love

You'd better remove them, because they could bring conflicts in the future.

**Comment by affektde** [comment]
thanks @taliptako <3
v6.x is a nice feature rich improvement, but I will wait for the feature like `$article->collectLikers();`
@antonkomarev is something already implemented, where I get the reacters collection, also by type would be awesome :-)

**Comment by taliptako** [comment]
@antonkomarev Thanks i did actually if you also add 5.8 support for version 5 that would be better i think you dont need to maintain it just merge the pull requests when needed its enough

**Comment by antonkomarev** [comment]
@taliptako I decided to stop improving old architecture because it's really hard or impossible to implement some features on it. I can accept PRs to support v5, but only if they wouldn't be breaking, because then it will require to make new major release what breaks SemVer, and package already gone forward.

`v5` branch created. If you want - make a PR to add 5.8 support to it and I release a new version.

**Comment by antonkomarev** [comment]
@affektde you can get reacters from reactant reactions:

```php
$reactions = $article
    ->getLoveReactant()
    ->reactions()
    ->get();

foreach ($reactions as $reaction) {
    dump($reaction->getReacter());
}
```

Or specify concrete type of the reactions:
```php
$likeReactions = $article
    ->getLoveReactant()
    ->reactions()
    ->where('reaction_type_id', ReactionType::fromName('Like')->getId())
    ->get();

foreach ($reactions as $reaction) {
    dump($reaction->getReacter());
}
```

**Comment by antonkomarev** [comment]
The main problem with adding Laravel 5.8 to Love 5.x, that it will require to drop PHP7.0 support or Laravel 5.5. And this is breaking changes. I think I will leave it as is, but turn off Laravel 5.5 tests, because they will fail all the time.

**Comment by antonkomarev** [comment]
[Laravel Love v5.2.1](https://github.com/cybercog/laravel-love/releases/tag/5.2.1) released with Laravel 5.8 support. @taliptako added to contributors list :+1:
